### ISSUE TITLE
Allow Limit to be Set on Query

### ISSUE THREAD

**Issue by wgranger** [zooniverse/zoo-stats-api-graphql#96]
A [graphQL query](https://github.com/zooniverse/touch-table-intermediate/pull/4/files#diff-7a2fd8595c9a5b458db868ff6280a622R29) needs the ability to set daily limits instead of defaulting to every day. Consider the following example:

```
    const query = `{
      statsCount(
        eventType: "classification",
        interval: "1 Day",
        projectId: "5733"
      ){
        period,
        count
      }
    }`
```

This will return daily counts for every day. Instead, the query should be able to accept a `limit: 1` tag to return classification counts for the current day.

The use case is for the touch table intermediate page, which will need to return daily classification counts for an entire project as well as daily classification counts for a single workflow.

**Comment by eatyourgreens** [comment]
If I set the interval to `1 Week` would `limit: 1` then mean only return one week? So `limit` is setting the ,max number of results that should come back.

Will the response be paged for large values of `limit`?

**Comment by trouille** [comment]
@camallen Because this request is in the context of our Touch Table Galaxy Zoo experience - which is launching in July, could you let us know the folllowing:

1) Can we continue to use the old stats API (so we can get the 'Total # of classifications from UScientist) until you have that stat available on the new stats API? 

2) Do you have a sense for timeline for when the new stats API will be able to provide the info and we can stop using the old stats API?

**Comment by trouille** [comment]
Here's the design for the intermediate page that Will is building (just for context):
https://invis.io/WFNHU46KV6J#/356092169_Desktop_HD

**Comment by camallen** [comment]
closed by #101
